### ISSUE TITLE
Can't find tags on inherited class in a different namespace

### ISSUE THREAD

**Issue by davewurtz** [EWSoftware/SHFB#354]
I have two classes in two different namespaces (BusinessObjects is my overall Namespace):

Namespace Standard
  Public Class A
    ''' <summary>
    ''' This is the summary"/>
    ''' </summary>
    Public Sub Foo
    End Sub
  End Class
End Namespace

Public Class A
  Inherits Standard.A
End Class

When I build the Sandcastle project, the help for the A.Foo method shows that the summary is missing ([Missing <summary> documentation for "M:BusinessObjects.Standard.A.Foo"]).  If I look at the .XML file, the method is defined for BusinessObjects.A, not BusinessObjects.Standard.A:

<member name="M:BusinessObjects.A.Foo">
<summary> This is the summary"/> </summary>
</member>

I believe the .XML file is correct.  How do I get Sandcastle to see the 'parent' namespace so it doesn't show the missing warning, but rather the "This is the summary" text?

Thanks.
Dave

**Comment by davewurtz** [comment]
I just realized my source code did not display correct in the post.  I'm attaching a 
[a.txt](https://github.com/EWSoftware/SHFB/files/522926/a.txt)
file...

**Comment by davewurtz** [comment]
I realized my comments, etc. didn't come through.  I'm attaching a ZIP file with a.vb and screen shots of the XML (helpXML.png) and the help file (helpXML.png).  I appreciate your help in advance!
Dave
[sandCastleHelpIssue_354.zip](https://github.com/EWSoftware/SHFB/files/522953/sandCastleHelpIssue_354.zip)

**Comment by EWSoftware** [comment]
It's hard to tell from those two snippets and a couple of screen shots.  At a guess, I'd say your XML comments file is out of date since the second class doesn't have a Foo method defined within it.  If it used to and it had comments, then that's what you're seeing.  I'd suggest cleaning the project to remove all output and then force a complete rebuild to get everything up to date.

**Comment by davewurtz** [comment]
I tried cleaning the project, which did delete the XML file.  After rebuilding the XML, it still shows the same.  Should the XML be generating the member name with the namespace (BusinessObjects.Standard) in my case?  Is this a problem with the XML generation or Sandcastle?  Ideally, I would like for BusinessObjects.A.Foo to be documented, not BusinessObjects.Standard.A.Foo.  Do I need to document the namespaces or the classes somehow.  I've tried a couple of ways but I can't seem to get it to work???
